quickwit/quickwit-directories/src/hot_directory.rs

```diff
@@ -205,14 +205,12 @@ impl StaticDirectoryCache {
         self.file_lengths.get(path).copied()
     }
 
-    /// return the files and their cached lengths
-    pub fn get_stats(&self) -> Vec<(PathBuf, usize)> {
+    pub fn get_file_lengths(&self) -> Vec<(PathBuf, u64)> {
         let mut entries = self
-            .slices
+            .file_lengths
             .iter()
-            .map(|(path, cache)| (path.to_owned(), cache.len()))
+            .map(|(k, v)| (k.clone(), *v))
```

Comment:
```suggestion
            .map(|(k, v)| (k.clone(), *v))
```

can we give a concrete name to k and v?